ompi/mca/osc/ucx/osc_ucx_component.c

```diff
@@ -21,6 +21,20 @@
     memcpy(((char*)(_dst)) + (_off), _src, _len); \
     (_off) += (_len);
 
+opal_mutex_t mca_osc_service_mutex = OPAL_MUTEX_STATIC_INIT;
+static void _osc_ucx_init_lock(void)
+{
+        if(mca_osc_ucx_component.enable_mpi_threads) {
+                    opal_mutex_lock(&mca_osc_service_mutex);
+                        }
```

Comment:
alignment is really strange

Reply:
@hoopoepg thanks, and done, I forced pushed the changes; however. I'm not sure if that's allowed or if it required a new commit